pyproject.toml

```diff
@@ -59,7 +59,7 @@ recommended = [
   "mkdocs-rss-plugin~=1.6"
 ]
 git = [
-  "mkdocs-git-committers-plugin-2~=1.1",
+  "mkdocs-git-committers-plugin-2~=2.4.1",
```

Comment:
Please limit this to `~=2.4`, not `~=2.4.1` – those have different meanings:

1. `~= 2.4`: means `>=2.4,<3`
2. `~=2.4.1`: means `>=2.4.1,<2.4`

We want 1., as we're only guarding against major version upgrades.

Reply:
Thinking about it, I'm not sure whether we need to change anything here, as 1.1 still works from our side, since [2.0 only changed the plugin's internals and/or configuration](https://github.com/ojacques/mkdocs-git-committers-plugin-2/releases), without necessary adaptions of our integration. We can relax the requirement to include everything above `>=1.1` up to `<3`. Thus, I think we should set it to `>=1.1,<3`.

Reply:
To sum up: we should always check what the absolute minimum requirements are and keep it at that as long as we can. This makes it easier for everyone downstream. My philosophy always is: as lax as possible, while guarding against major releases that might include breaking changes.

Reply:
Yes, that makes sense. I will change it

Reply:
Perfect, thanks!